pkg/api/controllers/health_controller_test.go

```diff
@@ -4,13 +4,14 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
 	"github.com/numary/ledger/pkg/api"
 	"github.com/numary/ledger/pkg/health"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/fx"
-	"net/http"
-	"net/http/httptest"
-	"testing"
 )
 
 func TestHealthController(t *testing.T) {
```

Comment:
This test is broken (already in "main" branch).
The fx options are configured but no fx app are started.

Reply:
Fixed